image-layout.md

```diff
@@ -3,47 +3,45 @@
 The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
 
-Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md) by:
+Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/bundle.md) by:
 
 * Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [manifest list](manifest-list.md#manifest-list)
 * [Applying the filesystem layers](layer.md#applying) in the specified order
-* Converting the [image configuration](config.md) into an [OCI Runtime Specification `config.json`](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md)
+* Converting the [image configuration](config.md) into an [OCI Runtime Specification `config.json`](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/config.md)
 
 # Content
 
-The image layout MUST contain two top level directories:
-
-- `blobs` contains content-addressable blobs.
-  A blob has no schema and should be considered opaque.
-- `refs` contains [descriptors][descriptors].
-  Commonly pointing to an [image manifest](manifest.md#image-manifest) or an [image manifest list](manifest-list.md#oci-image-manifest-list-specification).
-
-Both `blobs` and `refs` MAY be empty.
-
-The image layout MUST also contain an `oci-layout` file:
-
-- It MUST be a JSON object
-- It MUST contain an `imageLayoutVersion` field
-    - The `imageLayoutVersion` value will align with the OCI Image Specification version at the time changes to the layout are made, and will pin a given version until changes to the layout are required
-- It MAY include additional fields
+The image layout is as follows:
+
+- `blobs` directory
+    - Contains content-addressable blobs
+    - A blob has no schema and should be considered opaque
+    - Directory MUST exist and MAY be empty
+    - See [blobs](#blobs) section
+- `oci-layout` file
+    - It MUST exist
+    - It MUST be a JSON object
+    - It MUST contain an `imageLayoutVersion` field
+    - See [oci-layout file](#oci-layout-file) section
+    - It MAY include additional fields
+- `index.json` file
+    - It MUST exist
```

Comment:
This doesn't match your “OPTIONAL index.json” commit message and PR summaries.  You'll probably want to update for consistency.

Personally, I have no problem with making `index.json` optional (it feels like a bigger requirement than an empty `refs` directory for folks who have no references).